demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -103,10 +107,22 @@ def test_get_inputs():
 
     inputs, errors = get_inputs(playbook)
 
-    expected_inputs = [{'Name': 'InputA', 'Description': '', 'Default Value': 'Name',
-                        'Source': 'File', 'Required': 'Optional'},
-                       {'Name': 'InputB', 'Description': 'This is input b', 'Default Value': 'johnnydepp@gmail.com',
-                        'Source': '', 'Required': 'Required'}]
+    expected_query = '(type:ip or type:file or type:Domain or type:URL) -tags:pending_review ' \
+        'and (tags:approved_black or tags:approved_white or tags:approved_watchlist)'
+    expected_inputs = [
+        {
+            'Name': 'InputA', 'Description': '', 'Default Value': 'File.Name', 'Required': 'Optional'
+        },
+        {
+            'Name': 'InputB', 'Description': 'This is input b',
+            'Default Value': 'johnnydepp@gmail.com', 'Required': 'Required'
```

Comment:
i like jonny

Reply:
It wasn't my addition. It was there before my edits, so I can't take credit.